upgrading.md

```diff
@@ -0,0 +1,12 @@
+# Upgrade Procedures
+
+This document describes how registries should add support for new APIs and features.
+
+## Referrers API
+
+The Referrers API here is described by [Listing Referrers](spec.md#listing-referrers) and [end-12a](spec.md#endpoints).
+When registries add support for the Referrers API, this API needs to account for manifests that were pushed before the API was available using the [Referrers Tag Schema](spec.md#referrers-tag-schema).
```

Comment:
being in an "upgrade" file this implies that there are two methods to account for,  one is an upgrade.. the other a fallback/preexisting case.. and that implies one will eventually go away or is not preferred?

I think I see pros/cons for both cases... (graph included vs. performance) not sure if upgrade case and fallback/preexisting case are good names for the two cases. 

Is the only difference whether or not the image/artifact with a refers is explicitly listed in an image.index?